singlestoredb/apps/_dashboards.py

```diff
@@ -0,0 +1,44 @@
+import typing
+import urllib.parse
+
+from ._config import AppConfig
+from ._process import kill_process_by_port
+
+if typing.TYPE_CHECKING:
+    from plotly.graph_objs import Figure
+
+
+def run_dashboard(
+    figure: 'Figure',
+    kill_existing_server: bool = True,
```

Comment:
signature should just be run_dashboard(figure)

Reply:
removed the `host`, but keeping other default parameters as discussed.